src/javascript/app_2/Assets/Trading/Categories/components_trade_categories.jsx

```diff
@@ -0,0 +1,120 @@
+import PropTypes         from 'prop-types';
+import React             from 'react';
+import { localize }      from '_common/localize';
+
+// Templates are from Binary 1.0, it should be checked if they need change or not and add all of trade types
+
+const ComponentTradeCategories = ({ category }) => {
```

Comment:
maybe omit the word Component? i.e.: `TradeCategories`

Reply:
yeah better